lms/static/js/inline_analytics.js

```diff
@@ -378,6 +371,23 @@ window.InlineAnalytics = (function() {
     }
 
 
+    function convertNameToArray(choiceNameString) {
+        // Convert the choice name string to a list
+        choiceNameString = choiceNameString.substring(1, choiceNameString.length -1).replace(/'/g, "");
```

Comment:
Nit/style: I'm often a fan of the following convention for simple (often single character) string replacements:
```javascript
var output = haystack.split(needle).join(thread);
// or here...
choiceNameString = choiceNameString.substring(1).split("'").join('');
```

When I've benchmarked this in the past, the split/join approach was faster than regexs.